docker-compose.yml

```diff
@@ -9,11 +9,11 @@ services:
         published: 4200
     environment:
       PROTOCOL: 'http:'
-      HOSTNAME: 0.0.0.0
+      HOSTNAME: localhost
```

Comment:
Will this not affect the problems with AWS?

Reply:
the problem with sandbox is fixed, and this is only used for local test